src/tests/Home.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Home from '../components/Home';
+
+it('matches home snapshot', () => {
+  const tree = renderer.create(
+    <Home />,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- Great job so far @adarachel But currently, 3 of your tests are failing. So kindly ensure that all your tests are well written and pass successfully :100: 

![image](https://user-images.githubusercontent.com/102699958/232224298-3fd69589-7bd4-4be8-b8a1-7440473b902f.png)